perceval/backends/github.py

```diff
@@ -111,13 +140,29 @@ def fetch_from_cache(self):
 
         cache_items = self.cache.retrieve()
 
-        for raw_issues in cache_items:
-            issues = json.loads(raw_issues)
-            for issue in issues:
-                yield issue
-
+        issues = []
+        # In the cache first is the item with issues and next users data
+        # Read issues and all their users before processing issues
+        for raw_item in cache_items:
+            item = json.loads(raw_item)
+            if 'login' in item:
+                self._users[item['login']] = item
+            else:
+                next_issues = item
+                for issue in issues:
+                    # Add user data to the issue
+                    self._add_users_from_cache(issue)
+                    yield issue
+                issues = next_issues
+        for issue in issues:
+            # Add user data to the issue
+            self._add_users_from_cache(issue)
+            yield issue
 
 class GitHubClient:
+    """ Client for retieving information from GitHub API """
+
+    _users = {}  # users cache
```

Comment:
Should it be a class attribute? As I understand about how the class works, it makes more sense to change it by a instance attribute.